src/ui/toolbar.js

```diff
@@ -34,9 +34,11 @@ function Toolbar({ settings }) {
         }
       }}
     >
-      <a href={settings.serverUrl} title={t('toolbar-button-opencast')}>
-        <Icon icon={faPlayCircle} />
-      </a>
+      {settings.opencast && settings.opencast.serverUrl &&
```

Comment:
`settings && settings.opencast && settings.opencast.serverUrl  […]`

Reply:
But `settings` should always be an object (i.e. not `null` or `undefined`). If it's not, its an evil bug somewhere else. So I don't think it makes sense to check for that. `settings.opencast` on the other hand might very well be missing.